test/fixtureAsmJson.sol

```diff
@@ -1,4 +1,7 @@
 contract C {
+  // Leaving an empty line here intentionally
+
+
   function f() returns (bytes4) {
```

Comment:
no public?

Reply:
This is not compiled, it is just for formatting.

Reply:
I don't understand, there is assembly above. I thought it would be the test expectation.

Reply:
It is not compiled by the compiler, only used by the formatting tool.